src/borg/testsuite/archiver/diff_cmd_test.py

```diff
@@ -150,6 +169,15 @@ def get_changes(filename, data):
         # File unchanged
         assert not any(get_changes("input/file_unchanged", joutput))
 
+        # Do not show a 0 byte change for a file whose contents weren't modified.
+        unexpected = {"type": "modified", "added": 0, "removed": 0}
+        assert unexpected not in get_changes("input/file_touched", joutput)
+        if not content_only:
+            assert {"mtime"}.issubset({c["type"] for c in get_changes("input/file_touched", joutput)})
```

Comment:
If `mtime` changes due to `touch`, won't `ctime` change as well? This isn't wrong, but I was checking `ctime` as well because I expected it to change.

Reply:
ctime on windows is file creation time. guess it won't change as long as the file exists.

on posix it is inode change time and it changes for all metadata or data changes.

Reply:
How do you feel about this?

```python
expected_changes = "mtime"

if os.name in ('posix',):
    expected_changes += "ctime"
```

Reply:
good idea. we have some platformflags, yeah, guess i could check for "not windows".

Reply:
done.

Reply:
CI says it worked. \o/